docs/Users/Install.rst

```diff
@@ -6,7 +6,7 @@ platforms are Linux and Windows. *coala* is known to work on OS X as well.
 *coala* is tested against CPython 3.3, 3.4 and 3.5.
 
 In order to run *coala* you need to install Python. It is recommended,
-that you install Python3 >= 3.3 from http://www.python.org.
+that you install Python3 >= 3.3 from `http://www.python.org <https://www.python.org/downloads/>`_.
```

Comment:
now i've noticed, but i dont think it's okay to leave a website to be clickable and it to point somewhere else. at least give the full website from the start, people should know what they are clicking before doing it.